docker_cli/utils.py

```diff
@@ -0,0 +1,22 @@
+import random
+from string import ascii_lowercase, digits
+
+
+def generate_codename(k: int = 10) -> str:
```

Comment:
It ends up getting used by one of the functions later in `docker_cli` to generate unique temporary images, since a lot of these end up getting spun up in tests but the temp images are an implementation detail not meant to be accessed by tests or users.